linkinator.config.json

```diff
@@ -1 +1,11 @@
-{"recurse":true,"skip":["https://codecov.io/gh/googleapis/","www.googleapis.com","img.shields.io"],"silent":true,"concurrency":10}
\ No newline at end of file
+{
+  "recurse": true,
+  "skip": [
+    "https://github.com/googleapis/nodejs-network-management/blob/master/CHANGELOG.md",
```

Comment:
Instead of adding this here then needing to remember to remove it later, I have been just adding an empty CHANGELOG.md

Reply:
Won't this get automatically removed with owlbot?